twisted/plugins/multichain_crawler.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
```

Comment:
We've got the dispersy tracker and the tunnel helper units. I guess we are just lacking the tribler daemon?